src/test/java/ibm/jceplus/junit/openjceplusfips/multithread/TestSHA512_224.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright IBM Corp. 2023
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution.
+ */
+package ibm.jceplus.junit.openjceplusfips.multithread;
+
+
+import ibm.jceplus.junit.base.BaseTestSHA512_224;
+import ibm.jceplus.junit.openjceplus.Utils;
+
+public class TestSHA512_224 extends BaseTestSHA512_224 {
+
+    //--------------------------------------------------------------------------
+    //
+    //
+    static {
+        Utils.loadProviderTestSuite();
+    }
+
+    //--------------------------------------------------------------------------
+    //
+    //
```

Comment:
Similar comment here to remove empty comments.

Reply:
Removed.